client/components/search/index.jsx

```diff
@@ -319,7 +321,7 @@ class Search extends Component {
 			'is-open': isOpenUnpinnedOrQueried,
 			'is-searching': this.props.searching,
 			'is-compact': this.props.compact,
-			'has-focus': this.state.hasFocus,
+			'has-focus': ! this.props.hideFocus && this.state.hasFocus,
```

Comment:
What's the purpose behind this new property?

Reply:
The search component has a focus style by default when focused, we don't need that for the search in the new design.